android/app/src/main/java/com/microsoft/codepush/react/CodePush.java

```diff
@@ -147,8 +151,8 @@ public static String getBundleUrl(String assetsBundleFileName) {
 
         return currentInstance.getBundleUrlInternal(assetsBundleFileName);
     }
-
-    public String getBundleUrlInternal(String assetsBundleFileName) {
+    
+    private String getBundleUrlInternal(String assetsBundleFileName) {
```

Comment:
Should we keep this `public`? Otherwise, the logging [here](https://github.com/Microsoft/react-native-code-push/pull/275/files#diff-7a4664ad8b157a07a7a6cf0a7de26861R110) no longer makes sense.